src/site/content/en/secure/fixing-mixed-content/index.md

```diff
@@ -0,0 +1,240 @@
+
+If your site is published using a content management system,
+it is possible that links to insecure URLs are inserted when pages are published.
+For example, images may be included with a full URL rather than a relative path.
+You will need to find and fix these within the CMS content.
+
+### Fixing mixed content
+
+Once you've found mixed content in your site's source,
+you can follow these steps to fix it.
+
+If you get a console message that a resource request has been automatically upgraded from HTTP to HTTPS,
+you can safely change the `http://` URL for the resource in your code to `https://`.
+You can also check to see if a resource is available securely by changing `http://` to `https://` in the browser URL bar
+and attempting to open the URL in a browser tab.
+
+If the resource is not available via `https://`, you should consider one of the following options:
+
+* Include the resource from a different host, if one is available.
+* Download and host the content on your site directly, if you are legally allowed to do so.
+* Exclude the resource from your site altogether.
+
+Having fixed the problem,
+view the page where you found the error originally and verify that the error no longer appears.
+
+### Beware of non-standard tag usage
+
+Beware of non-standard tag usage on your site.
+For instance, anchor (`<a>`) tag URLs don't result in mixed content errors,
+as they cause the browser to navigate to a new page.
+This means they usually don't need to be fixed.
+However some image gallery scripts override the functionality of the `<a>`
+tag and load the HTTP resource specified by the `href` attribute into a lightbox display on the page,
+causing a mixed content problem.
+
+## Handle mixed content at scale
+
+The manual steps above work well for smaller websites;
+but for large websites or sites with many separate development teams,
+it can be tough to keep track of all the content being loaded.
+To help with this task, you can use content security policy
+to instruct the browser to notify you about mixed content and ensure that your pages never unexpectedly load insecure resources.
+
+### Content security policy
+
+[Content security policy](https://developers.google.com/web/fundamentals/security/csp/) (CSP)
+is a multi-purpose browser feature that you can use to manage mixed content at scale.
+The CSP reporting mechanism can be used to track mixed content on your site,
+and provide enforcement policies to protect users by upgrading or blocking mixed content.
+
+You can enable these features for a page by including the
+`Content-Security-Policy` or `Content-Security-Policy-Report-Only` header in the response sent from your server.
+Additionally you can set `Content-Security-Policy`
+(though **not** `Content-Security-Policy-Report-Only`) using a `<meta>` tag in the `<head>` section of your page.
+
+{% Aside %}
+Browsers enforce all content security policies that they receive.
```

Comment:
Nitpick: Since the other article acknowledges some users being on older browsers, I'd suggest replacing this with "Modern browsers enforce..."